include/renderer_gl/surfaces.hpp

```diff
@@ -79,7 +113,7 @@ struct ColourBuffer {
 struct DepthBuffer {
     u32 location;
     PICA::DepthFmt format;
-    OpenGL::uvec2 size; // Implicitly set to the size of the framebuffer
+    Math::uvec2 size; // Implicitly set to the size of the framebuffer
```

Comment:
Nowadays we have glm too btw for all vector/matrix/quaternion needs but the "old" vec implementation is fine here too since there's no math being really done.

(This is just an informational comment, nothing needs to change here)